weave

```diff
@@ -1717,13 +1773,26 @@ launch_router() {
             --no-restart)
                 RESTART_POLICY=
                 ;;
+            --awsvpc)
+                WEAVE_NO_FASTDP=1
+                AWSVPC_ARGS="--awsvpc"
+                AWSVPC=1
+                ;;
             *)
                 ARGS="$ARGS '$(echo "$1" | sed "s|'|'\"'\"'|g")'"
                 ;;
         esac
         shift
     done
     eval "set -- $ARGS"
+
+    create_bridge
+    docker_bridge_ip
```

Comment:
This is actually unnecessary because `create_bridge` calls it, but leaving it where it was would have avoided breaking line 1735.